src/pages/distroSettings/tabs/HostTab/getFormSchema.tsx

```diff
@@ -1,24 +1,34 @@
+import { css } from "@emotion/react";
 import { InlineCode } from "@leafygreen-ui/typography";
 import { GetFormSchema } from "components/SpruceForm";
-import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
 import {
-  Arch,
-  BootstrapMethod,
-  CommunicationMethod,
-  FeedbackRule,
-  HostAllocatorVersion,
-  OverallocatedRule,
-  Provider,
-  RoundingRule,
-  SshKey,
-} from "gql/generated/types";
+  AccordionFieldTemplate,
+  CardFieldTemplate,
+  FieldRow,
+} from "components/SpruceForm/FieldTemplates";
+import { size } from "constants/tokens";
+import { Arch, BootstrapMethod, Provider, SshKey } from "gql/generated/types";
+import {
+  architectureToCopy,
+  bootstrapMethodToCopy,
+  communicationMethodToCopy,
+  feedbackRuleToCopy,
+  hostAllocatorVersionToCopy,
+  linuxArchitectures,
+  nonWindowsArchitectures,
+  overallocatedRuleToCopy,
+  roundingRuleToCopy,
+  windowsArchitectures,
+} from "./constants";
 
 type FormSchemaParams = {
+  architecture: Arch;
   provider: Provider;
   sshKeys: SshKey[];
 };
 
 export const getFormSchema = ({
```

Comment:
Should we also try surfacing this error through the `validate()` function?

<img width="545" alt="Screenshot 2023-09-22 at 12 34 47 PM" src="https://github.com/evergreen-ci/spruce/assets/47064971/51dbd701-690b-42a3-a15a-4f5d1a4c2d51">